packages/base/api/base.api

```diff
@@ -48,36 +48,21 @@ public final class elide/runtime/LogLevel : java/lang/Enum {
 }
 
 public abstract interface class elide/runtime/Logger {
-	public abstract fun debug (Lkotlin/jvm/functions/Function0;)V
-	public abstract fun debug ([Ljava/lang/Object;)V
-	public abstract fun error (Lkotlin/jvm/functions/Function0;)V
-	public abstract fun error ([Ljava/lang/Object;)V
-	public abstract fun info (Lkotlin/jvm/functions/Function0;)V
-	public abstract fun info ([Ljava/lang/Object;)V
+	public fun debug (Lkotlin/jvm/functions/Function0;)V
+	public fun debug ([Ljava/lang/Object;)V
+	public fun error (Lkotlin/jvm/functions/Function0;)V
+	public fun error ([Ljava/lang/Object;)V
+	public fun info (Lkotlin/jvm/functions/Function0;)V
+	public fun info ([Ljava/lang/Object;)V
 	public abstract fun isEnabled (Lelide/runtime/LogLevel;)Z
 	public abstract fun log (Lelide/runtime/LogLevel;Ljava/util/List;Z)V
-	public abstract fun trace (Lkotlin/jvm/functions/Function0;)V
-	public abstract fun trace ([Ljava/lang/Object;)V
-	public abstract fun warn (Lkotlin/jvm/functions/Function0;)V
-	public abstract fun warn ([Ljava/lang/Object;)V
-	public abstract fun warning (Lkotlin/jvm/functions/Function0;)V
-	public abstract fun warning ([Ljava/lang/Object;)V
-}
```

Comment:
a little concerned about all this thrash, but as far as i can tell, this is a result of the plugin properly resolving defaults from interfaces now, due to the argument change in `kapt`. we'll see